nerfstudio/utils/writer.py

```diff
@@ -346,6 +362,29 @@ def write_config(self, name: str, config_dict: Dict[str, Any], step: int):
         self.tb_writer.add_text("config", str(config_dict))
 
 
+class CometWriter(Writer):
```

Comment:
Should it have the same decorator as the other writers? `@decorate_all([check_main_thread])`

Reply:
Thanks for the catch! added it!